js/ui/nav_bar.js

```diff
@@ -9,6 +9,11 @@ const NAVBAR_ITEM_CLASS = 'dx-nav-item';
 const NAVBAR_ITEM_CONTENT_CLASS = 'dx-nav-item-content';
 
 const NavBar = Tabs.inherit({
+    _init: function() {
+        this.callBase();
+        this._logDeprecatedWarning('W0000', { since: '20.1', alias: 'dxTabs' });
```

Comment:
Maybe it's better to move warning in a constructor? Just in case if the code will fall before widgets initialization.

Reply:
I will move this code to the constructors of widgets.